docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/managed-cli-example-create.md

```diff
@@ -408,6 +408,19 @@ ybm cluster encryption update-state \
 Successfully ENABLED encryption at rest for cluster my-sandbox
 ```
 
+## Scale your cluster
```

Comment:
Slot this in after one of the previously created clusters - the multi region one I suppose - and actually modify the previously created cluster (ie my-multi-region). This should be an example you can actually run, but frank-wren is never created